src/ServiceFactory/DoctrineDbalServiceFactory.php

```diff
@@ -0,0 +1,277 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Chubbyphp\DoctrineDbServiceProvider\ServiceFactory;
+
+use Chubbyphp\Container\Container;
+use Chubbyphp\Container\ContainerInterface;
+use Chubbyphp\DoctrineDbServiceProvider\Logger\DoctrineDbalLogger;
+use Chubbyphp\DoctrineDbServiceProvider\Registry\Psr\DoctrineDbalConnectionRegistry;
+use Doctrine\Common\Cache\ApcuCache;
+use Doctrine\Common\Cache\ArrayCache;
+use Doctrine\Common\Cache\Cache;
+use Doctrine\Common\EventManager;
+use Doctrine\DBAL\Configuration;
+use Doctrine\DBAL\DriverManager;
+use Doctrine\DBAL\Types\Type;
+
+final class DoctrineDbalServiceFactory
```

Comment:
@healerz i am working on a chubbyphp-container integration, what do you think about the usage?